### ISSUE TITLE
text under onscreen keyboard

### ISSUE THREAD

**Issue by guai** [microsoft/terminal#8983]
# Expected behavior

when onscreen keyboard is shown and terminal's window maximized then text in the terminal should not be printed under the kb, instead its working area should be shrinked to the visible area

# Actual behavior

the terminal does print text in the area I don't see that is covered with onscreen keyboard

**Comment by zadjii-msft** [comment]
Interesting. That's a good find. 

How this would play with split panes, I'm not sure. Say you've got a horizontal split in the terminal `[-]`. You bring up the onscreen keyboard - it entirely covers the bottom pane. What would we do then?

maybe related: #8623 - if we did have a scroll-forward (even just one page), then you could at least scroll forward to have the text be visible.

**Comment by Don-Vito** [comment]
No, bot.. don't close it.. it is important feature.. @zadjii-msft -help :blush:

**Comment by zadjii-msft** [comment]
Derp wait yea, this doesn't need author feedback - but it does need _some_ feedback I suppose. I'm not really sure how best to handle this scenario. That being said, it's something we should to eventually.

**Comment by DHowett** [comment]
I'm going to put this on the backlog, but it needs design.

**Comment by PankajBhojwani** [comment]
Does anyone know of an app that does this well? Trying to look for inspiration but it seems like Outlook, Word, Notepad all print text under the on-screen keyboard
